models/system/setting.go

```diff
@@ -88,7 +88,7 @@ func GetSettingNoCache(key string) (*Setting, error) {
 	if len(v) == 0 {
 		return nil, ErrSettingIsNotExist{key}
 	}
-	return v[key], nil
+	return v[strings.ToLower(key)], nil
```

Comment:
I have the suspicion that we lowercase to often at the moment.

Reply:
Yes, but are there better idea to reduce them?

Reply:
Extracting internal methods that assume the key is lowercase already?
However, I approved this PR as this is off-topic and simply a performance improvement.